src/main/content/_assets/js/javadocs.js

```diff
@@ -122,6 +125,9 @@ function addExpandAndCollapseToggleButtonForPackageFrame(contents, leftBottom) {
     // for string comparison.
     var header2_text = header2.text().replace('/\s/g',' ').trim();
     if(header2_text === "All Classes") {
+        // header2.css("position", "fixed");
+        // header2.css("top", "0");
+        // header2.css("width", "100%");
```

Comment:
Can you please remove the commented out code?

Reply:
Sure